apiary.apib

```diff
@@ -5834,47 +5835,43 @@ This kind of data is not kept forever. By default, all files are stored for **15
 using the `isPermanent` parameter. Files can also be tagged to categorize file uploads. When the `X-KBC-RunId` header is
 provided, it is stored as the file `runId`.
 
-Internally, all files are stored encrypted on [Amazon S3](http://docs.aws.amazon.com/AmazonS3/latest/dev/Welcome.html)
-and are private. When you access the file through the API, a URL with one hour expiration is generated. If you require
-a permanent URL, you can set the file as `public`.
+Internally, all files are stored encrypted and are private. When you access the file through the API,
+a URL with one hour expiration is generated.
 
-Large files may be split into slices (chunks). A sliced file must have the `sliced` parameter set to `true`. In that case
-the S3 key points to a **manifest file**. The manifest file contains the actual list of uploaded chunks. When uploading a sliced
+Large files may be split into slices (chunks). A sliced file must have the `isSliced` parameter set to `true`. In that case
+the provided key (blob) points to a **manifest file**. The manifest file contains the actual list of uploaded chunks. When uploading a sliced
 file, you have to use the provided key as a prefix for chunks and generate the manifest file with the list of uploaded chunks.
 When a CSV file is imported into Storage Tables, there is no difference between a normal and a sliced file.
 
 **Example:**
 
 Assume that the [upload method](#reference/files/upload-file/) returns the following S3 key `exp-180/11/files/2014/02/25/11085.import`.
 Then you can upload e.g. these files:
-- exp-180/11/files/2014/02/25/11085.import_001
-- exp-180/11/files/2014/02/25/11085.import_002
-- exp-180/11/files/2014/02/25/11085.import_003
+- exp-180/11/files/2014/02/25/11085.import001
+- exp-180/11/files/2014/02/25/11085.import002
+- exp-180/11/files/2014/02/25/11085.import003
 
 Then create the following manifest file:
 ```
 {
   "entries": [
-    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import_001"},
-    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import_002"},
-    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import_003"}
+    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import001"},
+    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import002"},
+    {"url":"s3://kbc-sapi-files-devel/exp-180/11/files/2014/02/25/11085.import003"}
   ]
 }
 ```
 
-And then, upload that manifest file to the key `exp-180/11/files/2014/02/25/11085.import_manifest`. See our
+And then, upload that manifest file to the key `exp-180/11/files/2014/02/25/11085.importmanifest`. See our
```

Comment:
URL k manifestu se generuje napevno tak, že se na konec přidá `manifest` bez oddělovače:
![CleanShot 2022-11-15 at 18 02 16@2x](https://user-images.githubusercontent.com/215660/201981264-c885dd16-9630-42a1-96b2-33bd19378486.jpg)
(Tak jsem změnil i ty url jednotlivých slisů v samplu, ať je to jednotný.)

Reply:
ono sa to v různýčh backendech ukládá různě třeba synapse generuje svoje soubory, ale takto je to OK.